CHANGELOG

```diff
@@ -1,3 +1,7 @@
+2016-04-22 Vitaliy Batichko <vbatichko@electric-cloud.com>
+  * 1.0.1: Project cleanup, added Decommission Environments
+              Temporarily disabled other procedures from plugin's step pickers
+
```

Comment:
Vitaliy, we haven't released v1.0.0 yet. This will be the first time we release the plugin so we can keep the version number as 1.0.0